src/test/HostActivationTests/GivenThatICareAboutMultilevelSDKLookup.cs

```diff
@@ -23,12 +23,15 @@ public class GivenThatICareAboutMultilevelSDKLookup
         private string _currentWorkingDir;
```

Comment:
Off topic discussion, the name of the file is hilarious. "Given" part should be an abbreviation of the test set up. In this case, Given mockDirsOfDifferentLevelAndTestProject